avocado/utils/network/interfaces.py

```diff
@@ -477,6 +477,22 @@ def is_available(self):
             log.debug(msg)
             return False
 
+    def is_bond(self):
+        """Check if interface is a bonding device.
+
+        This method checks if the interface is a bonding device or not.
+
+        rtype: bool
+        """
+        cmd = 'cat /proc/net/bonding/{}'.format(self.name)
+        try:
+            run_command(cmd, self.host)
```

Comment:
You are not concerned about the content here, because of that I would recommend to check if the file exist and is a file using the python api for that, also use as property since is a question. Something like this:

```python
@property
def is_bond(self):
    return os.path.isfile('/proc/net/bonding/{}'.format(self.name))
```

Reply:
I agree, we are not concerned with the content, but I would like to check if the interface is a bond interface on the remote side as well, hence `run_command`.

Reply:
Humm I see. In this case I will make my final comments.